website/blog/2021-02-22-jest-27.md

```diff
@@ -0,0 +1,75 @@
+So with some [tricks](https://github.com/facebook/jest/issues/9898), we've **sped up the initialization per test file** by roughly 70% compared to Jest 26. Note that you will almost certainly not see this big of a performance improvement on your project—you would need a lot of test files that each run very quickly to best notice this, and the overhead when using a [JSDOM environment](https://jestjs.io/docs/en/configuration#testenvironment-string) dwarfs any such improvement.
+
+In other news, the [native ESM support](https://github.com/facebook/jest/issues/9430) is progressing, but some major complexities, for instance around mocking, are still ahead of us, and we continue to observe the migration to ESM as a huge ecosystem effort, where Node and a lot of crucial tools and packages all have to rely on each other to deliver an overall compelling experience.
+
+<!-- TODO pending merge https://github.com/facebook/jest/pull/9351 -->
+
+We've also merged [a PR](https://github.com/facebook/jest/pull/9351) to be able to deal with test files symlinked into the test directory, a feature much wanted by users of [Bazel](https://bazel.build/).
+
+## Flipping defaults
+
+Up until now, if you were using Jest in its default configuration, you were—perhaps unknowingly—running some code forked many years ago from the test runner [Jasmine 2.0](https://jasmine.github.io/2.0/introduction) that provides test framework functions such as `describe`, `it`, and `beforeEach`. In 2017, [Aaron Abramov](https://github.com/aaronabramov) [initially wrote](https://github.com/facebook/jest/pull/3668) a **replacement for the Jasmine code** called `jest-circus`, meant to improve error messages, maintainability, and extensibility.  
+After years of large-scale use at Facebook, and of course in Jest itself, we are now confident that it is highly compatible with `jest-jasmine2` and should work in most environments with little to no migration work. There may be minor differences in execution order and strictness, but we expect no major upgrade difficulties other than for code relying on Jasmine-specific APIs such as `jasmine.getEnv()`. If you rely extensively on such APIs, you can opt back in to the Jasmine-based test runner by [configuring](https://jestjs.io/docs/en/configuration#testrunner-string) `"testRunner": "jest-jasmine2"`.
+
+Running tests in a [JSDOM environment](https://jestjs.io/docs/en/configuration#testenvironment-string) incurs a significant performance overhead. Because this was the default behavior of Jest unless otherwise configured up until now, users who are writing Node apps, for example, may not even know they are given an expensive DOM environment that they do not even need.  
```

Comment:
something about "plain javascript can run on node-like env, only when using browser APIs is jsom needed"? I.e. even _if_ you need JSDOM for some of your tests, you should probably use node as the default and turn on jsdom for the ones that need it

Reply:
Yeah that's a good idea to mention, I also think for projects where JSDOM is only required in some tests, Node as a performant default is good